lib/OpenQA/Utils.pm

```diff
@@ -743,6 +743,10 @@ sub human_readable_size {
     my ($size) = @_;
 
     my $p = ($size < 0) ? '-' : '';
+    $size = abs($size);
+    if ($size < 3000) {
+        return "$p$size Byte";
+    }
     $size = abs($size) / 1024.;
```

Comment:
You can remove the abs here.

Reply:
yeah, I noticed the same when reviewing the diff - but didn't want to force push and cause another round of travis failures before it's reviewed :)